confero-front/src/components/nav/Link.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import MaterialLink, { LinkProps } from "@material-ui/core/Link";
+import {
+  Link as RouterLink,
+  LinkProps as RouterLinkProps
+} from "react-router-dom";
+
+// Suggested Material-UI/ReactRouter integration from https://material-ui.com/guides/composition/#link
+const LinkWithRef = React.forwardRef<HTMLAnchorElement, RouterLinkProps>(
+  (props, ref) => <RouterLink innerRef={ref} {...props} />
+);
+
+const Link: React.FC<LinkProps & RouterLinkProps> = props => {
+  return <MaterialLink component={LinkWithRef} {...props} />;
```

Comment:
You have to do a little dance to get the Material UI link to render the React Router link with the right types. Luckily I had already done this on a previous project, so I just ported that over :grin: